apps/frontend/pages/404.tsx

```diff
@@ -0,0 +1,33 @@
+import link from 'next/link'
+import styles from '../styles/404.module.css'
+
+export default function Custom404() {
+  return (
+    <div className={styles.wrapper}>
+
+      <h1 className={styles.mainH}><b>404.</b></h1>
+
+      <a className={styles.properText}>Może szukałeś czegoś innego? Sprawdź to:</a>
+
+      <div className={styles.divlist}>
+
+      <ul className={styles.listmenu}>
+        <li>
+          <a className={styles.mainref} href="/">Strona Główna</a>
+        </li>
+
+        <li>
+          <a className={styles.news} href="/news">Aktualności</a>
```

Comment:
wrap `<a>` in `next/link` - https://nextjs.org/docs/api-reference/next/link

tldr; we use that to make transition between pages smoother; always use it